src/main/kotlin/purchase/domain/vo/CancelRequest.kt

```
@@ -0,0 +1,11 @@
+package purchase.domain.vo
+
+import java.math.BigDecimal
+
+data class CancelRequest(
+    val paymentKey: String,
+    val orderId: String,
+    val cancelReason: String,
+    val amount: BigDecimal
+) {
+}
```

detekt (1.23.7)

[warning] The class or object CancelRequest is empty.

(detekt.empty-blocks.EmptyClassBlock)